siliconcompiler/tools/verilator/compile.py

```diff
@@ -25,12 +25,22 @@ def setup(chip):
 
     chip.add('tool', tool, 'task', task, 'option', ['--cc', '--exe'],
              step=step, index=index)
-    chip.set('tool', tool, 'task', task, 'input', f'{design}.v',
-             step=step, index=index)
     chip.add('tool', tool, 'task', task, 'option', f'-o ../outputs/{design}.vexe',
              step=step, index=index)
 
 
+def runtime_options(chip):
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+
+    cmdlist = runtime_options_tool(chip)
+
+    for value in chip.find_files('input', 'hll', 'c', step=step, index=index):
+        cmdlist.append(value)
```

Comment:
should you add input,hll,c as a required field for this?

Reply:
good catch - it's not required to provide a C/C++ file to Verilator in this mode but I'll add the "if chip.valid(...) then require" pattern since that'll check that the filepath is valid if it's provided.